solutions/io.lionweb.mps.json/models/io.lionweb.mps.json.instance.mps2lionweb.mps

```diff
@@ -3619,6 +3624,25 @@
   </node>
   <node concept="312cEu" id="6VkSF6aF166">
     <property role="TrG5h" value="ClosureMps2LionWebConverter" />
+    <node concept="312cEg" id="3i3chebGTy_" role="jymVt">
+      <property role="TrG5h" value="exportUnconvertedParentIds" />
+      <node concept="3Tm6S6" id="3i3chebGTf5" role="1B3o_S" />
+      <node concept="10P_77" id="3i3chebGTvO" role="1tU5fm" />
+      <node concept="3clFbT" id="3i3chebGTNB" role="33vP2m" />
+      <node concept="z59LJ" id="3i3chebGVPG" role="lGtFl">
+        <node concept="TZ5HA" id="3i3chebGVPH" role="TZ5H$">
+          <node concept="1dT_AC" id="3i3chebGVPI" role="1dT_Ay">
+            <property role="1dT_AB" value="If true, exports parent node ids, even if parent is not converted." />
+          </node>
+        </node>
+        <node concept="TZ5HA" id="3i3chebGW5R" role="TZ5H$">
+          <node concept="1dT_AC" id="3i3chebGW5S" role="1dT_Ay">
+            <property role="1dT_AB" value="Defaults to false." />
```

Comment:
I thought we agreed that defaulting to exporting parent node IDs (even if unconverted) was the sensible default?

Reply:
You're right, fixed it (and added tests).